Release build failed for v0.13.0

https://github.com/hyperledger/firefly/runs/5191670680?check_suite_focus=true

```
Run make DOCKER_ARGS='\
./docker_build.sh \
  --label commit=ITHUB_SHA \
  --label build_date= \
  --label tag= \
  --tag ghcr.io/hyperledger/firefly:
--label commit=ITHUB_SHA --label build_date= --label tag= --tag ghcr.io/hyperledger/firefly:
FIREFLY_BUILDER_TAG=golang:1.16-alpine3.15
FABRIC_BUILDER_TAG=golang:1.16-alpine3.15
FABRIC_BUILDER_PLATFORM=linux/x86_64
SOLIDITY_BUILDER_TAG=ethereum/solc:0.8.11-alpine
BASE_TAG=alpine:3.15
UI_TAG=v0.5.0
UI_RELEASE=v0.5.0_8cb358c
invalid argument "ghcr.io/hyperledger/firefly:" for "-t, --tag" flag: invalid reference format
See 'docker build --help'.
make: *** [Makefile:[9](https://github.com/hyperledger/firefly/runs/5191865574?check_suite_focus=true#step:3:9)4: docker] Error [12](https://github.com/hyperledger/firefly/runs/5191865574?check_suite_focus=true#step:3:12)5
Error: Process completed with exit code 2.
```

--- a/.github/workflows/docker_main.yml
+++ b/.github/workflows/docker_main.yml
@@ -22,11 +22,11 @@ jobs:
 
       - name: Build
         run: |
-          make DOCKER_ARGS='\
+          make DOCKER_ARGS="\
             --label commit=$GITHUB_SHA \
             --label build_date=$(date -u +"%Y-%m-%dT%H:%M:%SZ") \
             --label tag=${{ steps.build_tag_generator.outputs.BUILD_TAG }} \
-            --tag ghcr.io/hyperledger/firefly:${{ steps.build_tag_generator.outputs.BUILD_TAG }}' \
+            --tag ghcr.io/hyperledger/firefly:${{ steps.build_tag_generator.outputs.BUILD_TAG }}" \
             docker
 
       - name: Tag release

--- a/.github/workflows/docker_release.yml
+++ b/.github/workflows/docker_release.yml
@@ -14,11 +14,11 @@ jobs:
 
       - name: Build
         run: |
-          make DOCKER_ARGS='\
+          make DOCKER_ARGS="\
             --label commit=$GITHUB_SHA \
             --label build_date=$(date -u +"%Y-%m-%dT%H:%M:%SZ") \
             --label tag=${GITHUB_REF##*/} \
-            --tag ghcr.io/hyperledger/firefly:${GITHUB_REF##*/}' \
+            --tag ghcr.io/hyperledger/firefly:${GITHUB_REF##*/}" \
             docker
 
       - name: Tag release

